packages/core/src/utils/parsers.tsx

```diff
@@ -70,6 +73,33 @@ export const isDidCommInvitation = (url: string) => {
   return false
 }
 
+export const parseMediatorInvitation = (url: string): Record<string, any> | null => {
+  if (!url.includes('c_i=')) return null
+
+  try {
+    const { c_i: encoded } = parse(url.split('?')[1] || '')
+    if (typeof encoded !== 'string') return null
+
+    let base64 = encoded.replace(/-/g, '+').replace(/_/g, '/')
+    while (base64.length % 4 !== 0) base64 += '='
```

Comment:
Another nit, but can we prefer multiline control flow statements? `if`s and loops I mean
```suggestion
    while (base64.length % 4 !== 0) {
      base64 += '='
    }
```

Just for readability